### PR TITLE
adding torture page

### PR DESCRIPTION
**Pull request by rubyimrie**

### CODE REVIEW

**Comment by rubyimrie** [comment]
have updated the info icon to match the others but this won't be visible until merged

File: pages/types-torture-different-residence.vue
Comment (Sephster) [inline pages/types-torture-different-residence.vue]:
```suggestion
              aria-label="Page Information"
```

File: pages/types-torture-different-residence.vue
Comment (Sephster) [inline pages/types-torture-different-residence.vue]:
```suggestion
            <span class="visually-hidden">Page Information</span>
```

File: pages/types-torture-different-residence.vue
Comment (Sephster) [inline pages/types-torture-different-residence.vue]: I am getting a strange bit of floating text that says page information. Suspect it is because of changes we've made in earlier branches but flagging to you to take a look. Thanks Ruby

**Review by Sephster — CHANGES_REQUESTED** [review]

File: pages/types-torture-different-residence.vue
Comment (rubyimrie) [reply pages/types-torture-different-residence.vue]: yeah this is because visibility hidden is in the layouts css which hasn't been merged in yet, should i add it in to this branch or wait till merge?

Comment (Sephster) [reply pages/types-torture-different-residence.vue]: ahh makes sense. I will merge in nuxt3-dev now as I'm now up to date. Will see how it looks after this. Thank you

Comment (Sephster) [reply pages/types-torture-different-residence.vue]: Looks good now so merging in. Thanks Ruby

**Pull request merged**
